services/cognito/app/controllers/metabase_token_controller.rb

```diff
@@ -12,6 +12,18 @@ def show
     end
   end
 
+  def show_identifier
+    identifier = params.delete(:identifier)
```

Comment:
Lets discuss this tomorrow. the idea is that we should support both id and identifier, so at this point we might not even need the custom endpoint.